recipes/colour-science/meta.yaml

```diff
@@ -0,0 +1,105 @@
+{% set name = "colour-science" %}
+{% set version = "0.3.8" %}
+{% set sha256 = "1234e3c8d4ac120030b605cbe1c578cb4dc2a9b9927a36efdd1bcd424c723290" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - scipy >=0.16.0
+
+  run:
+    - matplotlib >=1.3.1
+    - mock
+    - python
+    - scipy >=0.16.0
+    - six >=1.10.0
+
+test:
+  imports:
+    - colour
+    - colour.adaptation
+    - colour.adaptation.dataset
+    - colour.adaptation.tests
```

Comment:
We can only test `colour` if you prefer, I don't mind either way, I used *conda* generated skeleton which populated all that.